app/src/features/HomeScreen/index.tsx

```diff
@@ -1,9 +1,38 @@
-import React from "react";
+import { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { selectIsUserLoggedIn } from "../authentication/redux/selectors";
+import HomeScreenHeader from "./components/header";
 
 const HomeScreen = () => {
 
+    const [signInModal, setSignInModal] = useState<boolean>(false);
+    const [signUpModal, setSignUpModal] = useState<boolean>(false);
+
+    const isUserLoggedIn = useSelector(selectIsUserLoggedIn);
+
+    useEffect(() => {
+        if(isUserLoggedIn){
+            setSignInModal(false);
+            setSignUpModal(false);
+        }
+    }, [isUserLoggedIn])
+
+    const onSignUpLinkCicked = () => {
+        setSignInModal(false);
+        setSignUpModal(true);
+    }
+
     return (
-        <div>...Under construction</div>
+        <HomeScreenHeader
```

Comment:
Da se nadovezem na prethodni komentar oko header-a,
u tom slucaju home screen uopste ne bi drzao ove state-ove niti imao ikakve veze sa headerom vec bi on ovde renderovao neke druge stvari (koje jos nemamo) tipa ono 'We recommend', 'Upcoming screenings' itd itd